src/test/java/BaseTest.java

```diff
@@ -36,11 +41,12 @@ public static void chromeConfigs() {
     // Send a parameter for 'baseURL' specified in XML
     @Parameters({"baseURL"})
     // Make baseURL parameter optional, if it is null, then set it to something)
-    public void launchBrowser(@Optional String baseURL) {
+    public void launchBrowser(@Optional String baseURL) throws MalformedURLException {
         if (baseURL == null)
             baseURL ="https://bbb.testpro.io";
         // driver = new ChromeDriver();
-        driver = new FirefoxDriver();
+        // driver = new FirefoxDriver();
+        driver = pickBrowser();
         actions = new Actions(driver);
         // Make webdriver load the pages REALLY slow
 //        WebDriver augmentedDriver = new Augmenter().augment(driver);
```

Comment:
**Please delete the empty line!** «An Empty Line is a "Code Smell» 

**The short answer is,** _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._